posthog/api/feature_flag.py

```diff
@@ -58,10 +55,15 @@ def update(self, instance: FeatureFlag, validated_data: Dict, *args: Any, **kwar
             validated_key = validated_data.get("key", None)
             if validated_key:
                 FeatureFlag.objects.filter(key=validated_key, team=instance.team, deleted=True).delete()
+            self._update_filters(validated_data)
```

Comment:
Does this need to be called twice?

Reply:
It's called once per method - once in create, once in update.

Reply:
Oh wow that line break in dark mode is not clear on github review